docs/installation/server-config/index.rst

```diff
@@ -133,7 +128,7 @@ For more information, see the :ref:`Streams parameter reference <streams_develop
 cache.max.bytes.buffering
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The maximum number of memory bytes to be used for buffering across all threads. The default value in KSQL is ``10000000`` (~ 10 MB).
+The maximum number of memory bytes to be used for buffering across all threads participating in a KSQL query. The default value in KSQL is ``10000000`` (~ 10 MB).
```

Comment:
Does the above section not need a `ksql.` prefix to the prop name?

Also, I'm not sure the current text helps a user understand what this setting does. Again, I think it would help if this section summarized the implications of smaller / larger values.